### ISSUE TITLE
add more activations and loss functions to neural net classifier

### ISSUE THREAD

**Issue by spookieoli** [spookieoli/VreeDB#38]
add more activations and loss functions to neural net classifier
